src/test/java/com/google/jenkins/plugins/computeengine/integration/ConfigAsCodeTestIT.java

```diff
@@ -68,14 +67,39 @@ public void testWorkerCreated() throws Exception {
 
     // There should be a planned node
     assertEquals(1, planned.size());
-
     String name = planned.iterator().next().displayName;
 
     // Wait for the node creation to finish
     planned.iterator().next().future.get();
-
     Instance instance = client.getInstance(PROJECT_ID, ZONE, name);
+    assertNotNull(instance);
+  }
+
+  @Test
```

Comment:
Not necessarily related to this PR, but how do we wish to structure IT's for config as code. With each new feature we implement, will we always have to write an integration test for its configuration as code counterpart and should all these tests live in one file  (ConfigAsCodeTestIT).

Reply:
Im not sure if we needs JCasC integration test of each new functionality..
If we have IC test for that feature normal JCasC test should be enough..

Reply:
I think in this case sense it directly affects worker creation having test parity for JCasC is merited so as to catch any edge cases which may be introduced through the JCasC plugin. However, I agree in general that not all features need 100% test parity for JCasC.